tests/utils/test_lang_utils.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+from assistant_dialog_skill_analysis.utils.lang_utils import LanguageUtility
+
+
+class TestLangUtils(unittest.TestCase):
+    """Test for lang utils module"""
+
+    def setUp(self):
+        unittest.TestCase.setUp(self)
+        self.skill_file = open(
+            "tests/resources/test_workspaces/skill-Customer-Care-Sample.json", "r"
+        )
+
+    def test_en(self):
+        util = LanguageUtility("en")
+        sent = util.preprocess("boston resided on the east coast of us!")
+        self.assertTrue("!" not in sent)
+        sent = util.tokenize(sent)
+        self.assertTrue("resid" in sent)
```

Comment:
If you unit test these small functions, I would rather test "equality" of the whole sentence. That the output is the way how you want it: `self.assertEqual(sent,...)`

Because these partial tests like `self.assertTrue("resid" in sent)` could be missing problems. Imagine your stemmer function goes wrong and you will return both original form and stemmed for in the sentence, something like: `boston resided resid on the east...`. Your current test will pass and everything seems to work. That's why it's better to check the whole output